### ISSUE TITLE
Latest migration missing

### ISSUE THREAD

**Issue by maingoh** [modlinltd/django-advanced-filters#68]
It looks like one migrations is missing : 
```
Migrations for 'advanced_filters':
  0003_auto_20180430_1634.py:
    - Alter field created_at on advancedfilter
    - Alter field created_by on advancedfilter
    - Alter field groups on advancedfilter
    - Alter field title on advancedfilter
    - Alter field url on advancedfilter
    - Alter field users on advancedfilter
```

I can make a PR but I would need a new release on pypi ? Could someone do it ?

version: 1.0.7.1

Thank you,

**Comment by asfaltboy** [comment]
Hi @maingoh thanks for reporting the issue. Which Django version are you using?

**Comment by maingoh** [comment]
I edited my comment. I use the version 1.0.7.1

**Comment by asfaltboy** [comment]
That's the version of advanced-filters. But, what's your Django version?

**Comment by maingoh** [comment]
Oh sorry 1.9.13

**Comment by maingoh** [comment]
I can probably give you the content of this migration tomorrow if you want?

**Comment by asfaltboy** [comment]
Sure, that would help.

Thanks!

**Comment by maingoh** [comment]
It looks like it came from this commit https://github.com/modlinltd/django-advanced-filters/commit/b71d1e92e759ed07bb1ac02bf2261759ec2ff621#diff-f2cc63ea5bf7fcd3dbaeb32452043175
The verbose_name needs a new migrations. Here is my generated migration : 
```python
# -*- coding: utf-8 -*-
# Generated by Django 1.9.13 on 2018-05-02 14:41
from __future__ import unicode_literals

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('advanced_filters', '0002_advancedfilter_created_at'),
    ]

    operations = [
        migrations.AlterField(
            model_name='advancedfilter',
            name='created_at',
            field=models.DateTimeField(auto_now_add=True, null=True, verbose_name='Created at'),
        ),
        migrations.AlterField(
            model_name='advancedfilter',
            name='created_by',
            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='created_advanced_filters', to=settings.AUTH_USER_MODEL, verbose_name='Created by'),
        ),
        migrations.AlterField(
            model_name='advancedfilter',
            name='groups',
            field=models.ManyToManyField(blank=True, to='auth.Group', verbose_name='Groups'),
        ),
        migrations.AlterField(
            model_name='advancedfilter',
            name='title',
            field=models.CharField(max_length=255, verbose_name='Title'),
        ),
        migrations.AlterField(
            model_name='advancedfilter',
            name='url',
            field=models.CharField(max_length=255, verbose_name='URL'),
        ),
        migrations.AlterField(
            model_name='advancedfilter',
            name='users',
            field=models.ManyToManyField(blank=True, to=settings.AUTH_USER_MODEL, verbose_name='Users'),
        ),
    ]
```

**Comment by maingoh** [comment]
Can I make a PR with this migration ? I would love to see it in a new release :smiley:

**Comment by maingoh** [comment]
Also it seems it can be squashed directly into the previous migration https://stackoverflow.com/questions/26503826/why-does-django-make-migrations-for-help-text-and-verbose-name-changes

**Comment by asfaltboy** [comment]
Yes, certainly, PRs are more than welcome!

**Comment by asfaltboy** [comment]
Closed in #63
